notes-webapp/src/main/webapp/vue-app/notes/components/NoteTreeviewDrawer.vue

```diff
@@ -88,7 +89,7 @@
               </div>
             </v-col>
           </v-row>
-          <template v-if="home && !exportNotes && filter !== $t('notes.filter.label.drafts')" class="ma-0 border-box-sizing">
+          <template v-if="home && !exportNotes && showTree && filter !== $t('notes.filter.label.drafts')" class="ma-0 border-box-sizing">
```

Comment:
It is better to set showTree as a computed prop, to avoid handling its value everywhere